drape_frontend/drape_measurer.cpp

```diff
@@ -14,17 +14,15 @@ void DrapeMeasurer::StartBenchmark()
   using namespace std::chrono;
   m_isEnabled = true;
 
-  auto currentTime = std::chrono::steady_clock::now();
-
 #ifdef GENERATING_STATISTIC
-  m_startScenePreparingTime = currentTime;
+  m_startScenePreparingTime = steady_clock::now();
```

Comment:
But it's up to you. If there are no invariants that depend on the fact that start times must be equal, current approach is good enough.